providers/implementations/keymgmt/mlkem_kmgmt.c

```diff
@@ -580,11 +634,12 @@ const OSSL_DISPATCH ossl_mlkem768_keymgmt_functions[] = {
     { OSSL_FUNC_KEYMGMT_GEN_CLEANUP, (void (*)(void))mlkem_gen_cleanup },
     { OSSL_FUNC_KEYMGMT_DUP, (void (*)(void))mlkem_dup },
     /*
-     * TODO(ML-KEM) don't do for now, see https://github.com/openssl/private/issues/698
-     *  { OSSL_FUNC_KEYMGMT_IMPORT_TYPES, (void (*)(void))mlkem_imexport_types },
-     *  { OSSL_FUNC_KEYMGMT_EXPORT_TYPES, (void (*)(void))mlkem_imexport_types },
-     *  { OSSL_FUNC_KEYMGMT_IMPORT, (void (*)(void))mlkem_export },
-     *  { OSSL_FUNC_KEYMGMT_EXPORT, (void (*)(void))mlkem_import },
-     */
+     * TODO(ML-KEM): https://github.com/openssl/private/issues/698
+     * Export/import functionality has been partially implemented.
```

Comment:
Worthwhile stating here which things are missing?

Reply:
The encode/decode logic pretty much matches what I saw in other providers. But from https://github.com/openssl/private/issues/698:

> prove interoperability -- and for that we need reference to a standard (please fill in the above if you can: I didn't find anything, so if we want to do this without a standard (?), we need to agree on/how to do interop testing with other implementations).

I haven't done anything like this

Reply:
ACK. Suggest referencing to https://github.com/openssl/openssl/issues/25885 now (contains draft spec references).